src/tuner/nccl_ofi_tuner.c

```diff
@@ -21,7 +21,7 @@ ncclResult_t nccl_ofi_tuner_init(size_t nRanks, size_t nNodes, ncclDebugLogger_t
 	 */
 	if (getenv("NCCL_ALGO") || getenv("NCCL_PROTO")) {
 		NCCL_OFI_WARN("The tuner plugin can not be loaded when explicitly choosing an algorithm or protocol with NCCL_ALGO/NCCL_PROTO");
-		return ncclInvalidUsage;
```

Comment:
This should be in its own PR, and @rajachan should figure out how this even compiled.

Reply:
*scratches head*.  I did see compilation fail when setting up https://github.com/aws/aws-ofi-nccl/blob/9d8a467e7aca71d3545563e2923c1de234f788fe/src/nccl_ofi_api.c#L46; looking to see why it wasn't failing here.

Reply:
let's move this discussion over here: https://github.com/aws/aws-ofi-nccl/pull/359

Reply:
This merged.